examples/graphit/riscv.mk

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) 2019, University of Washington All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without modification,
```

Comment:
Do we need this now that we have one in cuda?

Reply:
No, this file can be deleted (and is on my list to remove when removing the dummy vec add test from the graphit directory as well).